app/histogram/src/lib.rs

```diff
@@ -79,13 +80,17 @@ impl Percentile {
 /// Deserializes the histogram from the given bytes.
 #[wasm_bindgen(js_name = deserializeHistogram)]
 pub fn deserialize_histogram(bytes: &[u8], width: isize) -> MiniHistogram {
+    if width == 0 {
```

Comment:
```suggestion
    if width <= 0 {
```

I think it should be `<=`? Since this width is an `isize` var.

Reply:
Good catch!